build/setupTestAppSettings.js

```diff
@@ -0,0 +1,47 @@
+// The goal of this file is to enable us to programmatically set the Razor OmniSharp plugin path. Typically when project system changes
+// happen we need to wait until OmniSharp consumes those changes to properly test them. This file enables us to locate the plugin and
+// then setup the test apps to use the new plugin path.
+
+var fs = require('fs');
+var path = require('path');
+
+function findInDir(directoryPath, needle) {
```

Comment:
Why call it `needle`? Never seen it used in this context before.

Reply:
Needle in a haystack 😄 lol just tried to figure out a different name than filename since I use it below.